pkg/antctl/raw/traceflow/command.go

```diff
@@ -195,13 +207,13 @@ func runE(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("error when retrieving Traceflow: %w", err)
 	}
 
-	if err := output(res); err != nil {
+	if err := output(res, cmd.OutOrStdout()); err != nil {
 		return fmt.Errorf("error when outputting result: %w", err)
 	}
 	return err
 }
 
-func newTraceflow(client kubernetes.Interface) (*v1alpha1.Traceflow, error) {
+func getNewTraceflow(client kubernetes.Interface) (*v1alpha1.Traceflow, error) {
```

Comment:
this renaming is not needed, the previous name is a more widely used style when creating an instance.